test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/SignCommandTests.cs

```diff
@@ -651,5 +653,48 @@ public void SignCommand_SignPackageWithUntrustedSelfIssuedCertificateInCertifica
                 }
             }
         }
+
+        [CIOnlyFact]
+        public async Task SignCommand_SignPackageWithUnsuportedTimestampHashAlgorithm_ShouldNotModifyPackageAsync()
+        {
+            var testServer = await _testFixture.GetSigningTestServerAsync();
+            var certificateAuthority = await _testFixture.GetDefaultTrustedCertificateAuthorityAsync();
+            var options = new TimestampServiceOptions() { SignatureHashAlgorithm = new Oid(Oids.Sha1) };
+            var timestampService = TimestampService.Create(certificateAuthority, options);
+
+            using (testServer.RegisterResponder(timestampService))
+            using (var directory = TestDirectory.Create())
+            {
+                var packageContext = new SimpleTestPackageContext();
+                var packageFile = packageContext.CreateAsFile(directory, fileName: Guid.NewGuid().ToString());
+
+                byte[] originalFile;
+                using (var packageStream = packageFile.OpenRead())
+                using(var ms = new MemoryStream())
+                {
+                    packageStream.CopyTo(ms);
+                    originalFile = ms.ToArray();
+                }
```

Comment:
The above 7 lines can be simplified to:

    var originalFile = File.ReadAllBytes(packageFile.FullName);